src/pdf_parser_opendataloader_hybrid.py

```
@@ -1,7 +1,81 @@
+
+
+def _ensure_server():
+    """Start the hybrid backend if it is not already running."""
+    global _server_process
+
+    if _is_server_running():
+        logger.info("Hybrid backend already running at %s", HYBRID_URL)
+        return
+
+    logger.info("Hybrid backend not running — starting opendataloader-pdf-hybrid ...")
+    _server_process = subprocess.Popen(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)